lib/cli.rb

```diff
@@ -0,0 +1,164 @@
+      puts "4. Change a Search Record"
+      puts "5. Exit program"
+
+      choice = STDIN.gets.chomp 
+
+      if choice == "1"
+        puts `clear`
+        Search.all.each do |search|
+          puts "Location: #{search.search_location}"
+        end 
+        puts
+        puts "Please type in a City to see the "+"Weather".colorize(:cyan)+", then press enter" 
+        user_input = STDIN.gets.chomp.strip
+        
+        actual_location = Search.all.find_by(search_location: user_input)
+        
+
+          if actual_location
+            weather_description =  Weather.find_by(search_id: actual_location.id)
+            puts "The "+"Weather".colorize(:cyan)+" in #{user_input} is "+"#{weather_description.description}".colorize(:cyan)
+            puts
+            puts "Wasn't expecting that weather, now were you."
+            puts 
+            saved_location = Search.create(search_location: user_input)
+
+            saved_location_in_history = UsersSearch.create(search_id: saved_location.id, user_id: $user.id)
+            
+          else
+            is_running = true            
+            puts
+            puts `clear`
+            puts "That City doesn't have "+"weather.".colorize(:cyan)+" WIERD. Please re-select Search "+"Weather.".colorize(:cyan) 
+          end 
+            
+          
+      elsif choice == "2" 
+        puts `clear`
+        puts 
+        UsersSearch.all.each do |history|
+          puts "Name - #{history.user.name} and Location - #{history.search.search_location}" 
+        puts 
+        end 
+        puts "Someone's being "+"drafty".colorize(:cyan)+"..."
+        puts 
+
+        # writing out my idea 
+        # if same user and same search location show up more than 3 times (iterate over searches) 
+        # then spit out puts "Notice how actual user is looking at specific location"
+  
+          # if specific_searches > 5
+          #   puts "Looks like someone has been searching #{search.search_location} a lot.."      
+          # else
+          #   puts "No particular place seems to be of interest to anyone."
+          # end  
+      elsif choice == "3" 
+        puts`clear`
+        puts 
+        UsersSearch.all.each do |history|
+          puts "ID - #{history.id}, Name - #{history.user.name} and Location - #{history.search.search_location}."  
+          end 
+          puts 
+          puts "Are you sure you want to delete a specific search? You're being stormy..." 
+          puts 
+        puts "Please type in the specific ID you wish to delete"
+        puts 
+        delete_select = STDIN.gets.chomp.strip
+          if UsersSearch.find_by(id: delete_select)
+            UsersSearch.delete(delete_select)
+            puts 
+            puts "You're shady. Like the "+"weather. ".colorize(:cyan)+"Get it?"
+            puts 
+            puts "You have deleted ID #{delete_select}. I hope you're happy."
+            puts 
+            is_running = true
+          else 
```

Comment:
Instead of using an if else chain, you should place the user in a loop until the output is what you expect.

You can assume the user got here because they wanted to delete something, don't send them back to the main menu. Think of the user experience, they always want to do more with less.